src/Components/AllCards/AllCards.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import "./AllCards.css";
+import {
+    useCreateCollection,
+    useUpdateSelectedCollection,
+    useDeleteSelectedCard,
+} from "../../Helpers/apiCalls";
+
+export default function AllCards(props) {
+    const {
+        pokemonCards,
+        setSelectedPokemon,
+        setPokemonCardData,
+        collections,
+        loggedInUser,
+    } = props;
+
+    const { mutate: createCollection } = useCreateCollection();
+
+    const { mutate: updateCollection } = useUpdateSelectedCollection();
+
+    const { mutate: deleteCard } = useDeleteSelectedCard();
```

Comment:
It looks like you don't use this mutation on this page anymore. Likely because you removed the delete card button. So you can remove this